### ISSUE TITLE
[Feature Request] Move buttons and text on mobile to the right

### ISSUE THREAD

**Issue by Imzxhir** [kuylar/lighttube#49]
### Description

I am not sure on the best way to explain this so to put it simply, I feel like button and text placements on mobile could move more to the right and not be stuck to the screen edge which just looks can look weird on some devices, here are some screenshots of this: https://rimgo.lunar.icu/a/HV8uLAe

### Describe the solution you'd like

The solution I would like is to move some of the text and buttons to the right so they can be seen more easilt and just overall look better in my opinion if they were moved to right a bit.

### Describe alternatives you've considered

I am not sure of any alternatives and have not considered any for now.

**Comment by kuylar** [comment]
Is this placement good or should we go with a different approach?
| Search | Settings |
|--|--|
|![Search page, filter button now uses more horizontal space](https://github.com/kuylar/lighttube/assets/52961639/4faee9e4-d635-4bb6-b101-49cbd8555219)|![Settings page, tabs are centered](https://github.com/kuylar/lighttube/assets/52961639/eb19f997-7689-4969-9ad1-a6d58540eeff) |

**Comment by Imzxhir** [comment]
I feel like the text and buttons look the same from the screenshot that I took.

**Comment by kuylar** [comment]
The "search filters" button now spans the whole width of the page, and the settings tabs are centered. Going to center the "Save Changes" button on the settings page now

**Comment by Imzxhir** [comment]
Yep, that looks better. By text and buttons, I mean the video titles and that whole part which could then fit the creators logo. Also, texts like the 'Theme' and 'Content Settings' could be moved to the right more. For buttons, I mean the 'Save Changes' button could be moved to the right.

**Comment by kuylar** [comment]
My current workspace. I think this is all the problems you noticed?
| Search | Settings |
|--|--|
|![Added padding to video titles & restored channel icons](https://github.com/kuylar/lighttube/assets/52961639/3043b51a-6efd-4541-b90c-cd2c80d2d8ec)|![Moved the "Save Changes" button to the center](https://github.com/kuylar/lighttube/assets/52961639/849b5453-a5c5-429f-8cad-b407b68b7e25)|

**Comment by Imzxhir** [comment]
Yep, that looks perfect.

**Comment by kuylar** [comment]
Implemented in [eb4783d](https://github.com/kuylar/lighttube/commit/eb4783ddadee25c668c8da5d9418f02cef8fcb34)
